operators/Makefile

```diff
@@ -338,6 +343,10 @@ ci: dep-vendor-only check-fmt generate check-local-changes unit integration e2e-
 # Let's use n1-standard-8 machine to have enough room for multiple pods on a single node.
 ci-e2e: ci-bootstrap-gke e2e
 
+# Run e2e tests in gke using custom operator image
+ci-e2e-rc: MANAGED_NAMESPACE=e2e
+ci-e2e-rc: ci-bootstrap-gke e2e-setup dep-vendor-only install-crds apply-operators e2e-docker-image e2e-run
```

Comment:
I don't understand why we need all this stuff vs. just reusing the existing `ci-e2e` target? What does it do differently?

Reply:
Fixed